### PR TITLE
Adapt to be able to upload to maven-central

### PR DESCRIPTION
**Pull request by mrunesson**
As follow up to https://github.com/GoogleCloudPlatform/DataflowJavaSDK/issues/97 this is an adaption of the join-library contrib module to be able to upload it to maven-central.

### CODE REVIEW

**Comment by mrunesson** [comment]
The contrib-module joinlibrary is now available via maven-central:
<dependency>
  <groupId>org.linuxalert.dataflow</groupId>
  <artifactId>google-cloud-dataflow-java-contrib-joinlibrary</artifactId>
  <version>0.0.3</version>
</dependency>

File: contrib/join-library/README.md [hunk 1]
```diff
@@ -22,4 +22,11 @@ Example how to use join-library:
     PCollection<KV<String, KV<String, Long>>> joinedPcollection =
       Join.innerJoin(leftPcollection, rightPcollection);
 
+Join-library can be found on maven-central:
+    <dependency>
```
Comment (dhalperi) [inline contrib/join-library/README.md]: add newline to fix the way this renders in Markdown?

File: contrib/join-library/pom.xml [hunk 2]
```diff
@@ -5,11 +5,33 @@
 
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>com.google.cloud.dataflow</groupId>
+  <groupId>org.linuxalert.dataflow</groupId>
   <artifactId>google-cloud-dataflow-java-contrib-joinlibrary</artifactId>
-  <name>Join library</name>
+  <name>Dataflow Join Library</name>
```
Comment (dhalperi) [inline contrib/join-library/pom.xml]: if you care, "Google Cloud Dataflow" rather than "Dataflow"

File: contrib/join-library/pom.xml [hunk 3]
```diff
@@ -20,9 +42,15 @@
...
+  <scm>
+    <connection>scm:git:git@github.com:GoogleCloudPlatform/DataflowJavaSDK.git</connection>
+    <developerConnection>scm:git:git@github.com:GoogleCloudPlatform/DataflowJavaSDK.git</developerConnection>
+    <url>https://github.com/GoogleCloudPlatform/DataflowJavaSDK/tree/master/contrib/join-library</url>
+  </scm>
+
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <google-cloud-dataflow-version>0.3.150227</google-cloud-dataflow-version>
+    <google-cloud-dataflow-version>1.3.0</google-cloud-dataflow-version>
```
Comment (dhalperi) [inline contrib/join-library/pom.xml]: try with 1.4.0, maybe upgrade? Or, `[1.0.0, 2.0.0)` for example

File: contrib/join-library/pom.xml [hunk 4]
```diff
@@ -5,11 +5,33 @@
...
+      <organizationUrl>http://www.linuxalert.org/</organizationUrl>
+      <roles>
+        <role>Project-Administrator</role>
+        <role>Developer</role>
+      </roles>
+      <timezone>+1</timezone>
+    </developer>
+  </developers>
+  <contributors>
+    <contributor>
```
Comment (dhalperi) [inline contrib/join-library/pom.xml]: can you please update this to match the `Authors.md` file? That has Google listed as well.

Would suggest copying [this section from root `pom.xml`](https://github.com/GoogleCloudPlatform/DataflowJavaSDK/blob/master/pom.xml#L50)

**Comment by dhalperi** [comment]
Great, thanks! Here are a few minor comments.

We are ready to merge whenever you are.

Thanks,
Dan

**Comment by mrunesson** [comment]
Thank you for the review. Review comments are fixed.

**Comment by davorbonaci** [comment]
LGTM.

**Pull request merged**